types/sinon-sandbox/sinon-sandbox-tests.ts

```diff
@@ -0,0 +1,5 @@
+import sinon = require('sinon');
+import sandbox = require('sinon-sandbox');
+import * as assert from "power-assert";
+
+assert.deepEqual(sandbox, sinon.createSandbox());
```

Comment:
```suggestion
import sandbox = require('sinon-sandbox');

sandbox; // $ExpectType SinonSandbox
```
code is not executed/evaluted on DT, just static, `tsc` compilation is part of CI tests, so `dtslint` assertions make more sense